docs/source/general/callback.rst

```diff
@@ -0,0 +1,41 @@
+########
+Callback
+########
+
+.. _callback:
+
+**************
+Flash Callback
+**************
+
+``FlashCallback`` are an extension of PyTorch Lightning :class:`~pytorch_lightning.callbacks.Callback`.
+
```

Comment:
maybe add more details:
"A callback is a self-contained program that can be reused across projects. You can use any of the following built in callbacks by ...."

Reply:
Done !